### DISCUSSION TITLE
posts/one-cup-of-tea/

### DISCUSSION THREAD

**Discussion by giscus** [sonnycalcr/sonnycalcr.github.io#7] [Announcements]
# posts/one-cup-of-tea/

关于 B 站的 sonnycalcr 这个帐号的事情，就在这个博客里面说。 今天与一位 B 站的网友私信时，我犯了一个习惯主义错误：一杯奶茶解决问题。 这好吗？我以前从来没有

https://sonnycalcr.github.io/posts/one-cup-of-tea/

**Comment by fak111** [comment]
个人理解：既然你也是群众的一员，那就不能拿群众一针一线。朋友同学除外。
要拿，可以谈面上的、广泛的东西给群众。但群众要是有特殊、个性的需求需要你满足？为什么不一杯奶茶？
就该拿，你提供了服务，也服务了社会。因为客户通过你，几乎没中介，其实已经以最便宜的方式获得服务了。

**Reply by sonnycalcr** [reply]
u said it. 不过，b 站的学生比较多，他们的经济毕竟不是自给自足，而且，学生时代的想法也比较受限，我觉得一定程度上是可以理解的。
